chdb/build.sh

```diff
@@ -123,8 +123,68 @@ if [ "$(uname)" == "Linux" ]; then
 fi
 
 # save the command to a file for debug
+echo ${LIBCHDB_CMD} > libchdb_cmd.sh.origin
```

Comment:
Maybe we can just pust static lib gen stuff into a separate `build_static.sh` fow now